docs/KRB5_TRACE.md

```diff
@@ -0,0 +1,36 @@
+# Setting KRB5_TRACE for gssproxy
+
+Enabling `KRB5_TRACE` output as described in `# man gssproxy.conf`:
+
+**At level 3 and above, KRB5_TRACE output is logged. If KRB5_TRACE was already set in the execution environment, trace output is sent to its value instead.**
+
+can be done either by including `KRB5_TRACE=/path/to/location` when executing gssproxy at the command line or by including a location within a drop file loaded by systemd. Systemd does not open `/dev/stderr` and instead creates a file descriptor for stderr when gssproxy is started. A drop file is necessary to record `KRB5_TRACE` information as gssproxy directs `KRB5_TRACE` to `/dev/stderr` at this time.
```

Comment:
Replace this:
`Systemd does not open `/dev/stderr` and instead creates a file descriptor for stderr when gssproxy is started.`
with
`Systemd uses unix sockets to redirect outputs to the journal, and this means /dev/stderr ends up pointing to a name that cannot be used as a path to open the stderr descriptor`

And later "at this time" -> "by default"

Reply:
Done. I had only mentioned "at this time" due to a potential change in how KRB5_TRACE is handled, but "by default" is more clear.